CMP1903_A1_2324/Testing.cs

```diff
@@ -15,6 +16,43 @@ internal class Testing
          * Use debug.assert() to make the comparisons and tests.
          */
 
+        public Game game = new Game();
         //Method
+        public void testrun()
+        {
+            //instantiate a game and pull the returned data from it
+            int[] testdata = game.roll3();
+            //inform the player of the current test step
+            Console.WriteLine("test game run");
+            //instantiate a new die
+            Die testDie = new Die();
+            //roll the test die
+            testDie.Roll();
+            //inform the user of the current test step
+            Console.WriteLine("test die run");
+            //debug test the die roll to ensure it is within bounds
+            Debug.Assert(testDie.Roll() > 0);
+            Debug.Assert(testDie.Roll() < 7);
```

Comment:
Can combine these into one statement using &&

Reply:
debugs combined in latest commit